typescript/sdk/src/deploy/HyperlaneAppChecker.ts

```diff
@@ -120,9 +121,11 @@ export abstract class HyperlaneAppChecker<
     }
   }
 
-  // TODO: Require owner in config if ownables is non-empty
-  async checkOwnership(chain: ChainName, owner: types.Address): Promise<void> {
-    const isOwnable = async (contract: Contract): Promise<boolean> => {
+  async ownables(chain: ChainName): Promise<{ [key: string]: Ownable }> {
+    const isOwnable = async (
```

Comment:
Nit: not totally convinced an async lambda + three `obj*` functions is simpler here than a good-ol for loop but not a big deal